src/com/lftechnology/java/training/sanish/stemleaf/Constants.java

```diff
@@ -0,0 +1,10 @@
+package com.lftechnology.java.training.sanish.stemleaf;
+
+public class Constants {
+	public static final int NUM_ZERO = 0;
+	public static final int MIN_NUM = 10;
+	public static final int MAX_NUM = 99;
+
+	private Constants() {
+	};
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)